packages/client/src/elements/Button/index.tsx

```diff
@@ -1,23 +1,24 @@
 import React from 'react';
-import { Link } from 'react-router-dom';
 
 import { ArrowIcon } from '../../icons';
+import { ThemeType } from '../../theme/Theme';
+import Link from '../Link';
 import Spinner from '../Spinner';
 import { IconContainer, StyledButton } from './Button.styled';
 import { ButtonProps } from './Button.types';
 
 const Button = ({
   children,
-  color = 'primary',
+  colorVariant = ThemeType.light,
   loading,
   to,
   disabled,
   withArrow,
   ...rest
 }: ButtonProps) => {
   const baseButton = (
-    <StyledButton color={color} disabled={loading ? true : disabled} {...rest}>
-      {loading && <Spinner color="primary" margin="0 8px 0 0" />}
+    <StyledButton colorVariant={colorVariant} disabled={loading ? true : disabled} {...rest}>
+      {loading && <Spinner margin="0 8px 0 0" />}
```

Comment:
put all styles to external file

Reply:
All default values we want to keep inside a component, not their styles.  If it comes to margin in out spinner we can't style component which isn't an instance of style-components. Obviously, we could wrap this in kinda div but this approach is much simpler and open for changes.